backend-project/small_eod/cases/models.py

```diff
@@ -46,14 +46,14 @@ class Case(TimestampUserLogModel):
     )
     notified_users = models.ManyToManyField(
         to=settings.AUTH_USER_MODEL,
-        related_name="notified_users",
+        related_name="notified_about_case_set",
```

Comment:
Dlaczego to potrzebujesz zmienić?

Reply:
hmm, widzę, ze branch się stworzył z innego brancha, nie dev, tylko z tego, gdzie pracowałem nad liczbą mnogą w cases. Zaraz zobaczę co z tym zrobić, ewentualnie przygotuję nowy pull request.